nestjs/src/app.controller.ts

```diff
@@ -0,0 +1,13 @@
+import { Controller, Get } from '@nestjs/common';
+import { AppService } from './app.service';
+
+@Controller()
```

Comment:
What processes these annotations in nestjs?

Reply:
Honestly not sure. I have never looked into it

Reply:
I guess I thought it would be compile time and there would be something in the PR. I guess it's runtime then? 🤷

Reply:
Actually `experimentalDecorators` adds metadata that it can use at runtime 👍

Reply:
That is good to know. Thanks